invenio_records_resources/services/custom_fields/errors.py

```diff
@@ -32,9 +33,10 @@ def __init__(self, field_name, given_namespace):
     @property
     def description(self):
         """Exception's description."""
-        return (
-            f"Namespace {self.given_namespace} is not valid for custom field "
-            f"{self.field_name}."
+        return _(
+            "Namespace %(given_namespace)s is not valid for custom field %(field_name)s.",
```

Comment:
should it that way?

i saw in other PR's that the form
```
_("Namespace %(given_namespace)s is not valid for custom field %(field_name)s.") % {"given_namespace": self.given_namespace, "field_name":self.field_name}
```
is there a reason it not to do here too?

Reply:
discussed and resolved